notebooks/WET/WET-003/WET-003-ComCam.ipynb

```diff
@@ -7,217 +7,757 @@
    "source": [
```

Comment:
<p>Would be good to explain that it is expected to work only at USDF  (I guess that based of the repo used)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>